pyvista/plotting/colors.py

```diff
@@ -875,6 +875,16 @@ def __hash__(self):  # pragma: no cover
         """Hash calculation."""
         return hash((self._red, self._green, self._blue, self._opacity))
 
+    def __getitem__(self, item):
+        """Support indexing for backward compatibility."""
+        if item < 0 or item > 3:
+            raise IndexError("Colors have only 4 channels.")
+        return self.float_rgba[item]
+
+    def __iter__(self):
+        """Support iterable for backward compatibility."""
+        return iter(self.float_rgba)
```

Comment:
Like you asked on the issue: should we include alpha here? E.g. did the background colour include it?

Reply:
For a Plotter's background color yes, for others probably not as there was not really one consistent representation (e.g. `parse_color` could return both float RGB and float RGBA tuples depending on the passed arguments). IMO when you index/iterate over a general `Color` instance you should be able to access every color channel, it seems arbitrarily limited otherwise to leave one color channel out. But I'm open for discussion and your opinions regarding this.

Reply:
I would agree that it would be more complete to include all of it. My only concern was that the original issue that this tries to fix was about handling backwards-compatibility with the background colour. But then the current implementation also works for that. This is enough for me :) Maybe document that these methods return the float RGBA representation?